src/pages/DetailsPage.js

```diff
@@ -66,6 +67,9 @@ const getPhoneNumber = (details) => {
 
 const DetailsPage = (props) => {
     const details = props.personalDetails[props.route.params.login];
+    if (!details) {
+        return <FullScreenLoadingIndicator />;
+    }
```

Comment:
Does seem like this would prevent the crash if we don't have the details. But it feels like a workaround that we only have to do because we don't have the details.

So the solution is to make sure we have the details - which brings us back to... it's a backend issue.